app/main/routes.py

```diff
@@ -34,7 +34,7 @@ def upload():
         response = post_ingest(excel_file, {"source_type": "tf_round_three"})
         response = response.json()
 
-        if response.get("status_code") == 200:
```

Comment:
We should use `response.status_code` here actually. The status code is returned as standard in the HTTP Response header. We then also put it in the Response body JSON. Better to use the header value.